hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/transaction/lock/StorageBasedLockProvider.java

```diff
@@ -276,9 +264,9 @@ private synchronized boolean isLockStillValid(StorageLockFile lock) {
    */
   @Override
   public synchronized boolean tryLock() {
-    assertHeartBeatManagerExists();
+    assertHeartbeatManagerExists();
```

Comment:
maybe we just use `ValidationUtils.checkState` instead

Reply:
I'm not familiar with what this does?

Reply:
I guess this could work, but I think we want to throw HoodieLockException, so not sure if IllegalStateException makes sense.